From 063e0311f4f394b4cdaf03b50121056c3479933d Mon Sep 17 00:00:00 2001
From: Tobias Fischer <info@tobiasfischer.info>
Date: Wed, 7 Dec 2022 07:01:04 +1000
Subject: [PATCH 08/10] fix strict prototype

---
 src_c/simd_blitters.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src_c/simd_blitters.h b/src_c/simd_blitters.h
index da0ecbb2..be7a05eb 100644
--- a/src_c/simd_blitters.h
+++ b/src_c/simd_blitters.h
@@ -61,7 +61,7 @@ void
 premul_surf_color_by_alpha_sse2(SDL_Surface *src, SDL_Surface *dst);
 
 int
-pg_has_avx2();
+pg_has_avx2(void);
 void
 blit_blend_rgba_mul_avx2(SDL_BlitInfo *info);
 void
-- 
2.38.1.windows.1

