From ec993dc74b8c79211ae0878e826b8bfffe1564ee Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Tue, 10 Jan 2023 14:50:59 +1100
Subject: [PATCH 10/10] don't -Werror on osx-arm

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 8a144bd2..e6b8767d 100644
--- a/setup.py
+++ b/setup.py
@@ -460,7 +460,7 @@ for e in extensions:
             and e.name not in ("pypm", "_sprite", "gfxdraw")
     ):
         # Do -Werror only on CI, and exclude -Werror on Cython C files and gfxdraw
-        e.extra_compile_args.append("/WX" if sys.platform == "win32" else "-Werror")
+        e.extra_compile_args.append("/WX" if sys.platform == "win32" else "")
 
 # if not building font, try replacing with ftfont
 alternate_font = os.path.join('src_py', 'font.py')
-- 
2.38.1.windows.1

